### PR TITLE
Configuration for yousign bundle 2.2

### PR DESCRIPTION
**Pull request by picoss**
| Q             | A
| ------------- | ---
| License       | MIT

Add recipe for Picoss Yousign bundle v2.2 that provides integration for
[Yousign API](http://developer.yousign.fr).

Yousign API offers a powerfull SOAP API to add a process of signature or archive document.

See https://yousign.com/ for more informations.

### CODE REVIEW

File: picoss/yousign-bundle/2.2/manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "bundles": {
+        "Picoss\\YousignBundle\\PicossYousignBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "#1": "To create a demo account, visit https://demo.yousign.fr/fr/public/api/registration",
+        "#2": "YOUSIGN_ENV should be 'demo' or 'prod'",
```
Comment (Nyholm) [inline picoss/yousign-bundle/2.2/manifest.json]: I don’t understand this comment.

File: picoss/yousign-bundle/2.2/manifest.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "bundles": {
+        "Picoss\\YousignBundle\\PicossYousignBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "#1": "To create a demo account, visit https://demo.yousign.fr/fr/public/api/registration",
```
[the remark this answers is not in the document]
Comment (picoss) [reply picoss/yousign-bundle/2.2/manifest.json]: I've removed the comments, and put them in the post-install.txt

File: picoss/yousign-bundle/2.2/manifest.json (hunk 1 above)
Comment (picoss) [reply picoss/yousign-bundle/2.2/manifest.json]: I've removed the comments, and put them in the post-install.txt

File: picoss/yousign-bundle/2.2/post-install.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<bg=blue;fg=white>                 </>
+<bg=blue;fg=white> Yousign Account </>
+<bg=blue;fg=white>                 </>
+
+  * You now need to create a yousign account
+    1. Visit <info>https://demo.yousign.fr/fr/public/api/registration</info>
+    2. Modify your YOUSIGN_API_KEY, YOUSIGN_USERNAME and YOUSIGN_PASSWORD config in <fg=green>.env</>
```
[the remark this answers is not in the document]
Comment (Nyholm) [reply picoss/yousign-bundle/2.2/post-install.txt]: Oops, you need to fix this and your PR will get merged.

**Pull request merged**